www/components/pages/Home/Inbox/Inbox.js

```diff
@@ -11,9 +19,19 @@ import styles from './Inbox.module.scss';
  * no meetings
  * @param {Function} refresh - a function that refreshes the meetings
  */
-export default function Inbox({ meetings, emptyMessage, refresh }) {
+export default function Inbox({
```

Comment:
Thoughts on changing the name of this component to meetings table?

Reply:
I prefer inbox but idc